bin/generic-download-tools.sh

```diff
@@ -17,8 +17,8 @@ getContent() {
 # $2 string Destination path to download the file to.
 download() {
 	if [[ `which curl` ]]; then
-		curl -s "$1" > "$2";
+		curl -sL "$1" > "$2";
```

Comment:
```suggestion
		curl -sL --proto '=https' --max-redirs 2 "$1" > "$2";
```
Here is a suggestion to improve security a bit. It restrict redirection to HTTPS only and allow only two redirect.
Ideally we would validate domain before redirect but it seems a bit overkill.